Assets/Scripts/UI/DraggableObject.cs

```diff
@@ -63,6 +72,8 @@ public virtual void OnBeginDrag(PointerEventData eventData)
 
     public virtual void OnDrag(PointerEventData eventData)
     {
+        transform.localScale = Vector3.Lerp(nonDraggingScale, draggingScale, (Time.time - startTime) * lerpSpeed); //Lerping
```

Comment:
With this code, the scale of the block only changes as it's being dragged. If you don't move your pointer at all, the lerp won't progress. Consider using the Update method instead.

The DraggableObject should also lerp back to its original size when the player lets go of it, so make sure to account for that too.